.circleci/config.yml

```diff
@@ -14,8 +14,34 @@ executors:
   golang:
     docker:
       - image: cimg/go:1.14
+  machine:
+    machine:
+      image: ubuntu-1604:201903-01
 
 commands:
+  setup:
+    steps:
+      - checkout
+      - restore_module_cache
```

Comment:
`save_module_cache` is not executed anywhere.

Reply:
Yeah that's getting addressed in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/310.